tests/spec/mesos_master/mesos_master_spec.rb

```diff
@@ -4,10 +4,54 @@
   it { should be_installed }
 end
 
-describe service('mesos-master') do
-  it { should be_running }
+describe "mesos master" do
+  describe service('mesos-master') do
+    it { should be_running }
+  end
+
+  describe command("curl --silent --show-error --fail --dump-header /dev/stderr --retry 2 http://#{host_inventory['hostname']}:5050/master/health") do
+    its(:stdout)  { should match /HTTP\/1.1 200 OK/ }
+  end
+  describe file('/var/lib/mesos') do
+    it { should be_directory }
+  end
+
+  describe file('/etc/mesos-master') do
+    it { should be_directory }
+
+    describe file('/etc/mesos-master/ip') do
+      it { should be_file }
+      its(:content) { should match /^172.31.1.1[123]/ }
+    end
+
+    describe file('/etc/mesos-master/quorum') do
+      it            { should be_file }
+      its(:content) { should contain '1' }
+    end
+  end
+
+  describe file('/etc/mesos/zk') do
+    it            { should be_file }
+    its(:content) { should contain('zk://172.31.1.1[123]:2181') }
```

Comment:
indeed this will break. For stuff thats dynamic (ip addresses) we should probably just test for the file, and maybe the content with some regex or something that is generic enough that its not hard-coded to a particular host so the test is portable to DO / AWS